packages/communication-react/review/communication-react.api.md

```diff
@@ -372,6 +373,11 @@ export interface CallingTheme {
     };
 }
 
+// @public
+export type CallOptions = {
+    showFloatingVideo?: "floatingLocalVideo" | "default";
```

Comment:
I'd expect a field named `showFloatingVideo` to be boolean. For an enum, I'd expect something like `videoMode`... but I can't think of a specific enough name for this enum. What values beyond these two do you anticipate in the future?

Also, if this is made boolean, it should be `enableFloatingLocalVideo`, for consistency with Chat options.